src/Libraries/Game.Server/Extensions/ItemExtensions.cs

```diff
@@ -65,6 +65,34 @@ public static uint GetMaxMagicWeaponDamage(this ItemData item)
         return item.GetMaxMagicWeaponBaseDamage() + item.GetAdditionalWeaponDamage();
     }
 
+    public static uint GetHairPartClientId(ItemInstance? itemInstance, EPlayerClass playerClass)
```

Comment:
I think you can make this an extension method because this class is an "extension class"